bigtable/cmd/emulator/cbtemulator.go

```diff
@@ -37,11 +38,18 @@ const (
 
 func main() {
 	grpc.EnableTracing = false
+	flag.Var(&emulatorInterceptorBuilder.LatencyTargets, "inject-latency", "Introduce gRPC latency for testing like \"ReadRows:p50:50ms\"")
+	flag.Var(&emulatorInterceptorBuilder.GrpcErrorCodeTargets, "inject-errors", "Introduce gRPC error codes for testing like \"ReadRows:10%:14\"")
```

Comment:
Can we keep all of the flags grouped together? Currently some flags are defined as globals while others are local vars in main

Reply:
We're defining the `emulatorInterceptorBuilder` in vars, but I think we have to wait until main to bind those vars using `flag.Var`